containers/kubernetes/install.sh

```diff
@@ -3,14 +3,19 @@
 set -ex
 
 # Install packages without dependencies
-if [ -z "$VERSION" ] && [ -z "$OFFLINE" ]; then
+if [ ! -z "$CENTOS" ] && [ -z "$VERSION" ] && [ -z "$OFFLINE" ]; then
     eval $(/container/scripts/get-version-env.sh)
 fi
 
-/container/scripts/install-rpms.sh --nodeps cockpit-bridge-
-/container/scripts/install-rpms.sh -a noarch --nodeps cockpit-shell-
-/container/scripts/install-rpms.sh cockpit-ws-
-/container/scripts/install-rpms.sh --nodeps cockpit-kubernetes-
+if [ -z "$CENTOS" ]; then
+	/container/scripts/install-rpms.sh --nodeps cockpit-bridge-
+	/container/scripts/install-rpms.sh -a noarch --nodeps cockpit-shell-
+	/container/scripts/install-rpms.sh cockpit-ws-
+	/container/scripts/install-rpms.sh --nodeps cockpit-kubernetes-
+else
+	INSTALL_PKGS="cockpit cockpit-bridge cockpit-shell cockpit-ws cockpit-kubernetes"
```

Comment:
cockpit-bridge and cockpit-kubernetes need to be installed without installing any of their dependencies.